smoke/_init_frr.sh

```
@@ -193,17 +176,21 @@ set_srv6_route() {
+    local frr_ip route
     if [[ "$prefix" == *:* ]]; then
 	    frr_ip="ipv6"
+	    route="route6"
     else
 	    frr_ip="ip"
+	    route="route4"
     fi
 
     # ----- build CLI & Grout forms ----------------------------------------
     local seg_frr   ; IFS=/ ; seg_frr="${sids[*]}"       # SID/SID/…
     local seg_space ; IFS=' ' ; seg_space="${sids[*]}"   # SID SID …
```

Shellcheck (0.11.0)

[warning] seg_space appears unused. Verify use (or export if used externally).

(SC2034)